### DISCUSSION TITLE
Welcome to chancejs Discussions!

### DISCUSSION THREAD

**Discussion by victorquinn** [chancejs/chancejs#505] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wlaeri** [comment]
Hi! My name is Will and I'm working on [Version 2.0](https://github.com/chancejs/chancejs/pull/550). If anyone is interested in working with TypeScript, we would love [your help](https://github.com/chancejs/chancejs/discussions/559) getting V2 out the door :)

**Reply by victorquinn** [reply]
Thanks for leading the charge here @wlaeri !

**Comment by vimil-saju** [comment]
I was wondering if chance library is tree shakeable. webpack does not seem to remove the module from bundling even though its only referenced from code that is within 
`if (process.env.NODE_ENV === 'development') {`
